pkg/mount/mountutil_test.go

```diff
@@ -70,13 +70,13 @@ func TestGetMountAttr(t *testing.T) {
 	}{
 		"sda4 mounted at /": {
 			"/dev/sda4",
-			DeviceMountAttr{MountPoint: "/", FileSystem: "ext4"},
+			DeviceMountAttr{MountPoint: []string{"/"}, FileSystem: "ext4"},
```

Comment:
Can you add a test with a single partition mounted at 2 paths.

Reply:
done